### ISSUE TITLE
Manual Connection (QR Code Scanning)

### ISSUE THREAD

**Issue by Shreemanarjun** [Shreemanarjun/flutter_sharez#16]
- Add a feature to add qr code scanning works 
- on scanned check the ip & port by connectng them to a common route ("Check server")
- if you got succesful response navigate to Device Page

**Comment by priyaranjan12345** [comment]
please assign to me I want to work on this

**Comment by Shreemanarjun** [comment]
Yes. i have assigned this to you. Let me know ,if any help needed.

**Comment by Abd-ul-Hannan** [comment]
qr scaning is not working when click on qrcode option so black screen come and still this error come 
Another exception was thrown: Cannot hit test a render box with no size.

**Comment by Shreemanarjun** [comment]
/gemini Check what can be done here with the steps to fix.

**Comment by Shreemanarjun** [comment]
@gemini-cli /review

**Comment by Shreemanarjun** [comment]
Fixed in v2
